### PR TITLE
Added in width to be inserted from meta

### PR DESCRIPTION
**Pull request by claytonnarcis**
This allows custom size rather than depending on actual image size

### CODE REVIEW

**Comment by mrvnklm** [comment]
@chaseconey Could you merge? Cheers

File: composer.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "chaseconey/nova-external-image",
-  "description": "A Laravel Nova field.",
+  "name": "claytonnarcis/nova-external-image-with-width",
```
Comment (chaseconey) [inline composer.json:2]: I can merge this up, but you will have to undo these changes

**Review by chaseconey — CHANGES_REQUESTED** [review]
Just need to revert the `composer.json` and we should be good to go.

**Comment by chaseconey** [comment]
Closing in favor of #4

**Pull request closed without merging**